pkg/plugin/ipam.go

```diff
@@ -63,10 +63,13 @@ func ExecAdd(plugin string, netconf []byte) (*Result, error) {
 	if os.Getenv("CNI_COMMAND") != "ADD" {
 		return nil, fmt.Errorf("CNI_COMMAND is not ADD")
 	}
+	if plugin == "" {
+		return nil, fmt.Errorf(`Name of IPAM plugin is missing. Specify a "type" field in the "ipam" section`)
+        }
```

Comment:
The formatting is off. Please always use [gofmt](http://golang.org/cmd/gofmt/) before contributing code.